### ISSUE TITLE
Problem adding the PDF export plugin

### ISSUE THREAD

**Issue by mfr-itr** [dradis/dradis-ce#137]
### Steps to reproduce

With `gem 'dradis-pdf_export', '~> 3.6', github: 'dradis/dradis-pdf_export'`, it says that html_export doesn't have a 3.6 version ([since it's 3.2.1](https://github.com/dradis/dradis-pdf_export/blob/master/lib/dradis/plugins/pdf_export/gem_version.rb#L10-L12)), and `~> 3.2` gives

```
Bundler could not find compatible versions for gem "dradis-plugins":
  In Gemfile:
    dradis-calculator_dread (~> 3.6) was resolved to 3.6.0, which depends on
      dradis-plugins (~> 3.0)

    dradis-calculator_dread (~> 3.6) was resolved to 3.6.0, which depends on
      dradis-plugins (~> 3.0)

    dradis-html_export (~> 3.6) was resolved to 3.6.0, which depends on
      dradis-plugins (~> 3.6)

    dradis-html_export (~> 3.6) was resolved to 3.6.0, which depends on
      dradis-plugins (~> 3.6)

    dradis-html_export (~> 3.6) was resolved to 3.6.0, which depends on
      dradis-plugins (~> 3.6)

    dradis-pdf_export (~> 3.2) was resolved to 3.2.1, which depends on
      dradis-plugins (~> 3.4.1)

    dradis-projects (~> 3.6) was resolved to 3.6.0, which depends on
      dradis-plugins (~> 3.6.2)
```

Is this a problem with my setup or should this plugin be updated?

### System configuration
**Dradis version**:

CE 3.6.0

**Ruby version**:

2.3.3

**OS version**:

Kali latest

**Comment by laim-user** [comment]
I have the same problem. Configuration: CE 3.6.0, ruby 2.2.3

**Comment by rachkor** [comment]
Hi @mfr-itr + @laim-user, I'm Rachael with the Dradis support team. The PDF exporter is currently in need of some love to get it compatible with the latest updates. 
Would you be interested in contributing a PR to help get `dradis-pdf_export` compatible with `dradis-plugins` 3.6? https://github.com/dradis/dradis-pdf_export
Otherwise, stay tuned for updates. We'll let you know here as soon as we have updates on this.

**Comment by rachkor** [comment]
PDF Export fixed in v3.7
